### PR TITLE
new file:   packages/oranger/oranger.3.0.1/opam

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
There seems to be a missing dependency:
```

#=== ERROR while compiling oranger.3.0.0 ======================================#
# context              2.0.9 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://github.com/UnixJunkie/oranger/archive/v3.0.0.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/oranger.3.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p oranger -j 31
# exit-code            1
# env-file             ~/.opam/log/oranger-5451-0a5907.env
# output-file          ~/.opam/log/oranger-5451-0a5907.out
### output ###
# File "src/dune", line 11, characters 22-28:
# 11 |  (libraries dolog cpm molenc minicli oranger line_oriented))
#                            ^^^^^^
# Error: Library "molenc" not found.
# Hint: try:
#   dune external-lib-deps --missing --no-config --root . --ignore-promoted-rules --default-target @install --always-show-command-line --promote-install-files --release --only-packages oranger -p oranger --profile release -j 31 @install
```

**Comment by UnixJunkie** [comment]
ok, I just added it.

**Comment by kit-ty-kate** [comment]
The build now fails with:
```
#=== ERROR while compiling oranger.3.0.0 ======================================#
# context              2.0.9 | linux/x86_64 | ocaml-base-compiler.4.11.2 | pinned(https://github.com/UnixJunkie/oranger/archive/v3.0.0.tar.gz)
# path                 ~/.opam/4.11/.opam-switch/build/oranger.3.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p oranger -j 31
# exit-code            1
# env-file             ~/.opam/log/oranger-16303-0a5907.env
# output-file          ~/.opam/log/oranger-16303-0a5907.out
### output ###
#       ocamlc src/.RFR.eobjs/byte/rFR.{cmi,cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.11/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.RFR.eobjs/byte -I /home/opam/.opam/4.11/lib/batteries -I /home/opam/.opam/4.11/lib/bst -I /home/opam/.opam/4.11/lib/bytes -I /home/opam/.opam/4.11/lib/cpm -I /home/opam/.opam/4.11/lib/dolog -I /home/opam/.opam/4.11/lib/line_oriented -I /home/opam/.opam/4.11/lib/minicli -I /home/opam/.opam/4.11/lib/molenc -I /home/opam/.opam/4.11/lib/num -I /home/opam/.opam/4.11/lib/ocaml/threads -I /home/opam/.opam/4.11/lib/re -I /home/opam/.opam/4.11/lib/seq -I src/.oranger.objs/byte -no-alias-deps -o src/.RFR.eobjs/byte/rFR.cmo -c -impl src/RFR.ml)
# File "src/RFR.ml", line 75, characters 18-37:
# 75 |     | [] -> raise Parany.End_of_input
#                        ^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Parany
```

**Comment by UnixJunkie** [comment]
Ok, added missing parany dep.

**Comment by kit-ty-kate** [comment]
Adding the dependency doesn't do anything as the dependency isn't listed in your dune files

**Comment by UnixJunkie** [comment]
Ok, I will correct that and issue a new tag.

**Comment by UnixJunkie** [comment]
should be OK now

File: packages/oranger/oranger.3.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+]
+install: [
+  ["cp" "ranger-0.9.11/cpp_version/build/ranger" "%{bin}%/ml_rf_ranger"]
+]
+depends: [
+  "dune" {>= "1.0.0"}
+  "conf-cmake" {build}
+  "re"
+  "dolog" {>= "6.0.0"}
+  "batteries"
```
Comment (mseri) [inline packages/oranger/oranger.3.0.1/opam:23]:
```suggestion
  "batteries" {>= "3.0.0"}
```
This is when BatString.split_on_char was introduced

**Comment by UnixJunkie** [comment]
@mseri thanks for the batteries lower bound; I upstream that

**Comment by UnixJunkie** [comment]
I don't know what to change more.

**Comment by UnixJunkie** [comment]
I'll check that. L.unfold_exn is a BatList function. Maybe, a lower bound
on batteries is needed.

**Comment by mseri** [comment]
Perfect! Thanks a lot

**Pull request merged**